roles/scala/tasks/activator.yml

```diff
@@ -11,14 +11,14 @@
   get_url: url={{activator_download_url}}
     dest=/tmp/{{activator_archive_file}}
     force=no
-  sudo: no
+  sudo: yes
 
 - name: Unpack activator
-  command: unzip {{activator_archive_file}} chdir=/tmp/
-  sudo: no
+  command: unzip {{activator_archive_file}} chdir=/tmp/ creates={{activator_archive_name}}
+  sudo: yes
```

Comment:
Sorry for my lack of English ability, but I'd like to ask why you changed the parameter of `sudo` option from `no` to `yes`?
Is any deep reason?